openedx/core/djangoapps/content_tagging/rest_api/v1/views.py

```diff
@@ -71,7 +66,8 @@ def get_queryset(self):
         else:
             queryset = get_taxonomies(enabled)
 
-        return queryset.prefetch_related("taxonomyorg_set")
+        # Prefetch tag_set so we can serialize the tag counts
+        return queryset.prefetch_related("taxonomyorg_set", "tag_set")
```

Comment:
We also need to prefetch the org because we match `org_short_name.`
```suggestion
        return queryset.prefetch_related("taxonomyorg_set_org", "tag_set")
```

Reply:
Yep, that drops off 2 queries, thanks!

I'm considering using the RequestCache with the rules to avoid re-querying where we can, but want to use that as last resort.